tasks/main.yml

```diff
@@ -627,6 +627,193 @@
         __mssql_conf_setting: "network forceencryption"
         __mssql_conf_setting_value: "{{ '1' if mssql_tls_enable else 'unset' }}"
 
+- name: Configure Active Directory authentication
+  vars:
+    __mssql_kinit_user: >-
+      {{ ad_integration_user }}@{{ ad_integration_realm
+      | upper }}
+  when: mssql_ad_configure | bool
+  block:
+    - name: Join to {{ ad_integration_realm }} realm
+      vars:
+        ad_integration_realm: "{{ ad_integration_realm }}"
+        ad_integration_password: "{{ ad_integration_password }}"
+      include_role:
+        name: linux-system-roles.ad_integration
+
+    - name: Ensure the adutil package
+      package:
+        name: adutil
```

Comment:
Does `adutil` have a dependency on the package that provides `klist`?  On Fedora 37 klist is provided by `krb5-workstation`

Reply:
Yes, adutil have a dependency on krb5-workstation
```
Installing:
 adutil                                    x86_64                          1.0.014-1                              packages-microsoft-com-prod                          6.4 M
Installing dependencies:
 krb5-workstation                          x86_64                          1.18.2-21.el8                          rhel                                                 957 k
 libkadm5                                  x86_64                          1.18.2-21.el8                          rhel                                                 187 k
```